pkg/auth/handler/webapp/authflowv2/settings_identity_view_email.go

```diff
@@ -116,4 +119,27 @@ func (h *AuthflowV2SettingsIdentityViewEmailHandler) ServeHTTP(w http.ResponseWr
 		h.Renderer.RenderHTML(w, r, TemplateWebSettingsIdentityViewEmailHTML, data)
 		return nil
 	})
+
+	ctrl.PostAction("remove", func() error {
+		requested := r.Form.Get("q_login_id")
+
+		// Page should be unable to display if q_login_id is not given. But just to be sure, we double check here.
+		if requested == "" {
+			result := webapp.Result{RedirectURI: "/web/not_found"}
+			result.WriteResponse(w, r)
+			return nil
+		}
```

Comment:
1. Define a json schema for remove from
2. Put the identity id in the form as a hidden input instead of getting from query string